src/components/AnswerSection.tsx

```diff
@@ -224,67 +178,57 @@ const AnswerSection = ({
     }
   }
 
-  const volumeIconColor =
-    result === '' && !secondAttempt ? COLORS.lunesBlackUltralight : isActive ? COLORS.lunesRed : COLORS.lunesRedDark
-
-  const volumeIconStyle = [styles.volumeIcon, (result !== '' || secondAttempt) && !isActive && styles.shadow]
-
   return (
-    <View style={styles.container}>
-      <Popover isVisible={isPopoverVisible} setIsPopoverVisible={setIsPopoverVisible} ref={touchable}>
-        <PopoverContent />
-      </Popover>
-
-      <TouchableOpacity
-        testID='volume-button'
-        disabled={result === '' && !secondAttempt}
-        style={volumeIconStyle}
-        onPress={() => handleSpeakerClick(document?.audio)}>
-        <VolumeUp fill={volumeIconColor} />
-      </TouchableOpacity>
-
-      <View
-        testID='input-field'
-        ref={touchable}
-        style={[
-          styles.textInputContainer,
-          {
-            borderColor: getBorderColor()
-          }
-        ]}>
-        <TextInput
-          style={styles.textInput}
-          placeholder={secondAttempt ? 'Neuer Versuch' : 'Wort mit Artikel eingeben'}
-          placeholderTextColor={COLORS.lunesBlackLight}
-          value={input}
-          onChangeText={text => setInput(text)}
-          editable={result === ''}
-          onFocus={() => setIsFocused(true)}
-          onBlur={() => setIsFocused(false)}
+    <>
+      <AudioPlayer document={document} disabled={result === '' && !secondAttempt} />
```

Comment:
Is it on purpose that the AudioPlayer is not wrapped in the container view?

Reply:
Yes, for the styling.